core/src/main/java/io/fabric8/maven/core/service/openshift/ImageStreamService.java

```diff
@@ -230,4 +238,21 @@ private String findTagSha(OpenShiftClient client, String imageStreamName, String
             throw new MojoExecutionException("Could not find a tag in the ImageStream " + imageStreamName);
         }
     }
+
+    private boolean isDate1AfterDate2(String date1, String date2) {
```

Comment:
Also the method doesn't access instance/member variables which is good indication that you may want to turn this into a Comparator .. Please note my java is rusty and dated.